src/components/NextMeetup.tsx

```diff
@@ -1,82 +1,96 @@
-            <span className="block text-4xl">
-              6:30pm AEST | The Precinct |{" "}
-              <a
-                className="underline underline-offset-4"
-                href="https://www.meetup.com/bitcoinbrisbane/events/293340730/"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                RSVP
-              </a>
-            </span>
-          </div>
-          <div className="hidden lg:flex flex-col gap-y-4">
-            <p className="mt-4 text-4xl">
-              Checkout our other events @
-              <a
-                className="underline underline-offset-4"
-                href="https://www.meetup.com/bitcoinbrisbane/"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                meetup.com/bitcoinbrisbane
-              </a>
-            </p>
-          </div>
-        </div>
+export const NextMeetup = () => {
+  const [events, setEvents] = useState([]);
+  const [error, setError] = useState(null);
+
+  function convertDate(date){
+    const splitDate = date.split("-")
+    const finalDate = `${splitDate[2]}/${splitDate[1]}/${splitDate[0]}`
+
+    return finalDate
+  }
+
+  function convertTime(time){
+    const splitTime = time.split(":")
+    const finalTime = `${splitTime[0]}:${splitTime[1]}`
+
+    return finalTime
+  }
+
+  useEffect(() => {
+    axios
+      .get("http://localhost:1337/api/events?populate=*")
+      .then(({data}) => {
+        setEvents(data.data);
+      })
+      .catch((error) => {
+        setError(error);
+      });
+  }, []);
+
+  if (error) {
+    return <div> An error occurred: {error.message}</div>;
+  }
+
+  return (
+    <section className="w-full mt-12 px-6">
+      <h2 className="font-black title text-5xl lg:text-9xl w-full text-center text-white">
+        NEXT MEETUP
+      </h2>
+      <div className="w-full mt-10 lg:mt-24 font-hand lg:flex">
+        <ul>
+          {events.toReversed().map((event) => (  
+            <li key={event.id} className="mt-12 lg:mt-36">
+              <div className="grid grid-cols-12 flex items-center justify-center w-full">
+                <div className="col-start-3 col-span-4 lg:flex lg:flex-col">
+                  <h3 className="text-3xl lg:text-6xl leading-[1.5rem] relative top-1">
+                    {event.attributes.name}
+                  </h3>
+                  <div className="hidden lg:flex flex-col gap-y-4">
+                    <p className="mt-4 text-2xl">
+                      {event.attributes.description}
+                    </p>
+                    <span className="block text-4xl text-red-700">
+                      {convertDate(event.attributes.date)} | {convertTime(event.attributes.time)} AEST | {event.attributes.venue} |{" "}
+                      <a
+                        className="underline underline-offset-4"
+                        href="https://www.meetup.com/bitcoinbrisbane/events/293340730/"
```

Comment:
this might go to a wrong event.  Lets change to the meetup site as we dont have the event id

```suggestion
                        href="https://www.meetup.com/bitcoinbrisbane/"
```